testcafe/tests/pages/photos-model.js

```diff
@@ -51,19 +39,47 @@ export default class Page {
     this.allPhotos = Selector('[class*="pho-photo"]').find(
       '[class*="pho-photo-item"]'
     )
+
+    // Photo fullscreen
+    this.photoFull = Selector('[class*="pho-viewer-imageviewer"]').find('img')
+    this.photoNavNext = Selector('[class*="pho-viewer-nav--next"]')
+    this.photoNavNextBtn = this.photoNavNext.find(
+      '[class*="pho-viewer-nav-arrow"]'
+    )
+    this.photoNavPrevious = Selector('[class*="pho-viewer-nav--previous"]')
+    this.photoNavPreviousBtn = this.photoNavPrevious.find(
+      '[class*="pho-viewer-nav-arrow"]'
+    )
+    this.photoBtnClose = Selector('[class*="pho-viewer-toolbar-close"]').find(
+      '[class*="c-btn"]'
+    )
+  }
+
+  async waitForLoading() {
+    const startTime = Date.now()
+    const timeout = 5000
+    while (await this.loading.exists) {
```

Comment:
Why ? waitForLoading() should just be something like : 

const waitForLoading = expect(Selector(this.loading).to.be.false);

No?